src/main/java/md/miller1995/Dealership/services/users/UserService.java

```diff
@@ -0,0 +1,27 @@
+package md.miller1995.Dealership.services.users;
+
+import md.miller1995.Dealership.models.dto.UserAuthDTO;
+import md.miller1995.Dealership.models.entities.UserAuthEntity;
+
+public interface UserService {
+
+    /**
+     * This method is used to register a new user in the database
+     *
+     * @author Anton Nirca
+     * @since  12/05/2023
+     *
+     * @param userAuthDTO
+     */
+    void registerUser(UserAuthDTO userAuthDTO);
```

Comment:
Create and Update methods, in most cases, should return the resource created/updated in the API. In this way, the caller has a confirmation not by the HTTP status code 200/201, but also he can check if the fields were saved properly.